ssz/utils.py

```diff
@@ -0,0 +1,25 @@
+from ssz.sedes import (
+    boolean,
+)
+
+
+def is_sedes(obj):
+    """
+    Check if `obj` is a sedes object.
+    A sedes object is characterized by having the methods
+    `serialize(obj)` and `deserialize(serial)`.
+    """
+    return hasattr(obj, 'serialize') and hasattr(obj, 'deserialize')
+
+
+def infer_sedes(obj):
+    '''
+    Try to find a sedes objects suitable for a given Python object.
+    '''
+    if is_sedes(obj.__class__):
```

Comment:
This is something we only get once we have containers, so I'd remove it for the time being.